packages/components/Button/src/ButtonTokens.ios.ts

```diff
@@ -10,75 +10,72 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
       width: '100%',
     },
     medium: {
-      padding: globalTokens.size80,
+      paddingHorizontal: globalTokens.size120,
       borderWidth: globalTokens.stroke.width10,
-      iconSize: 16,
+      borderRadius: globalTokens.corner.radius80,
+      minHeight: 40,
+      iconSize: 20,
       focused: {
         borderWidth: 0,
-        padding: globalTokens.size100,
       },
       hasContent: {
         minWidth: 96,
-        paddingHorizontal: globalTokens.size160 - globalTokens.stroke.width10,
         hasIconAfter: {
-          spacingIconContentAfter: globalTokens.size100,
+          spacingIconContentAfter: globalTokens.size80,
         },
         hasIconBefore: {
-          spacingIconContentBefore: globalTokens.size100,
+          spacingIconContentBefore: globalTokens.size80,
         },
         focused: {
-          paddingHorizontal: globalTokens.size160,
+          paddingHorizontal: globalTokens.size120,
         },
       },
     },
     small: {
-      padding: globalTokens.size80 - globalTokens.stroke.width10,
+      paddingHorizontal: globalTokens.size60,
       borderWidth: globalTokens.stroke.width10,
+      borderRadius: globalTokens.corner.radius80,
+      minHeight: 28,
       iconSize: 16,
       focused: {
         borderWidth: 0,
-        padding: globalTokens.size80,
       },
       hasContent: {
         minWidth: 64,
-        minHeight: 24,
-        paddingHorizontal: globalTokens.size120 - globalTokens.stroke.width10,
+        minHeight: 28,
         hasIconAfter: {
-          spacingIconContentAfter: globalTokens.size80,
+          spacingIconContentAfter: globalTokens.size40,
         },
         hasIconBefore: {
-          spacingIconContentBefore: globalTokens.size80,
+          spacingIconContentBefore: globalTokens.size40,
         },
         focused: {
-          paddingHorizontal: globalTokens.size120,
+          paddingHorizontal: globalTokens.size60,
         },
       },
     },
     large: {
-      padding: globalTokens.size120 - globalTokens.stroke.width10,
+      paddingHorizontal: globalTokens.size160,
       borderWidth: globalTokens.stroke.width10,
       iconSize: 20,
+      borderRadius: globalTokens.corner.radius120,
+      minHeight: 52,
       focused: {
         borderWidth: 0,
-        padding: globalTokens.size120,
       },
       hasContent: {
         minWidth: 96,
-        paddingHorizontal: globalTokens.size160 - globalTokens.stroke.width10,
         hasIconAfter: {
-          spacingIconContentAfter: globalTokens.size100,
+          spacingIconContentAfter: globalTokens.size80,
         },
         hasIconBefore: {
-          spacingIconContentBefore: globalTokens.size100,
+          spacingIconContentBefore: globalTokens.size80,
         },
         focused: {
           paddingHorizontal: globalTokens.size160,
         },
       },
     },
-    rounded: {
-      borderRadius: globalTokens.corner.radius40,
-    },
```

Comment:
Do we just not have a rounded button on iOS?

Reply:
Rounded button is the default, the others are variants, so we still have rounded, this is per the docs.